tf_encrypted/protocol/securenn/securenn.py

```diff
@@ -353,9 +353,38 @@ def relu(self, x):
 
     :param PondTensor x: Input tensor.
     """
-    with tf.name_scope('relu'):
-      drelu = self.non_negative(x)
-      return drelu * x
+
+    def relu(x):
```

Comment:
consider renaming (`actual_relu`, `sub_relu`, ?) to avoid any confusion

Reply:
done